### ISSUE TITLE
Remove the nginx configuration from the getting started chapter

### ISSUE THREAD

**Issue by DelazJ** [qgis/QGIS-Documentation#7519]
Page URL: https://docs.qgis.org/3.22/en/docs/server_manual/getting_started.html

IMHO "getting started" sections should help users quickly onboard in a minimal way and bring them to a kind of result. This may apply to the Windows & Linux Apache sections. OTOH, the nginx section provides many different ways to do a server configuration and that may confuse beginners very quickly. I suggest we moved that section out of the "getting started" section, and maybe find a way to include it in the [config page](https://docs.qgis.org/3.22/en/docs/server_manual/config.html) as an alternative/advanced way to do things

**Comment by pblottiere** [comment]
I think I agree. BTW, it's probably related to https://github.com/qgis/QGIS-Documentation/issues/7244. Setting the web server is not directly related to QGIS Server so the training manual could probably be a good vessel.

**Comment by DelazJ** [comment]
Yeah... I first thought I would add my comment at the end of #7244 but realized that the scope of the nginx content was not totally clear to me, and in the training we should bring people to a result (something I do not find in the nginx section).
